packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -56,6 +57,7 @@ export default class {
     }
     this._xapis = { __proto__: null }
     this._xo = xo
+    this._noEventsTimeout = parseDuration(noEventsTimeout)
```

Comment:
Have you tried it? It should not be working…

Reply:
You've not answered this question.

Reply:
I just tested it :)